sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -115,6 +115,36 @@ def _forward_pass(self, activations):
 
         return activations
 
+    def _forward_pass_fast(self, X):
+        """Predict using the trained model
```

Comment:
Please add a short description indicating how this differs from `_forward_pass` e.g.

> This is the same as _forward_pass but does not record the activations of all layers and only returns the last layer's activation.

Reply:
I have added the text you suggested. Thanks!